RNs-4.10.20:adds zstream release (#47186)
openshift/openshift-docs: release_notes/ocp-4-10-release-notes.adoc

==== Upgrading

To upgrade an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.

[id="ocp-4-10-20"]
=== RHBA-2022:5172 - {product-title} 4.10.20 bug fix update

Issued: 2022-06-28

{product-title} release 4.10.20 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:5172[RHBA-2022:5172] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5171[RHBA-2022:5171] advisory.

Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:

link:https://access.redhat.com/solutions/6964873[{product-title} 4.10.20 container image list]

[id="ocp-4-10-20-bug-fixes"]
==== Bug fixes

> * Previously, Bond Container Network Interface (CNI) version 1.0 was not compatible with the Multus Container Network Interface (CNI) plug-in, which resulted in Bond-CNI IP address management (IPAM) results to improperly populate the `network-status` annotation. With this update, IPAM and Bond-CNI now supports Bond-CNI 1.0. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084289[*BZ#2084289*])

Reviewer: ```suggestion
* Previously, Bond Container Network Interface (CNI) version 1.0 was not compatible with the Multus Container Network Interface (CNI) plug-in. Consequently, the Bond-CNI IP address management (IPAM)  improperly populated the `network-status` annotation. With this update, IPAM and Bond-CNI now supports Bond-CNI 1.0. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084289[*BZ#2084289*])
```

> * Previously, Bond Container Network Interface (CNI) version 1.0 was not compatible with the Multus Container Network Interface (CNI) plug-in. Consequently, the Bond-CNI IP address management (IPAM) results to improperly populated the `network-status` annotation. With this update, IPAM and Bond-CNI now supports Bond-CNI 1.0. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084289[*BZ#2084289*])

Reviewer: Thought I'd add it as a comment for when you get back. 

```suggestion
* Previously, Bond Container Network Interface (CNI) version 1.0 was not compatible with the Multus Container Network Interface (CNI) plug-in. Consequently, the Bond-CNI IP address management (IPAM) improperly populated the `network-status` annotation. With this update, IPAM and Bond-CNI now supports Bond-CNI 1.0. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084289[*BZ#2084289*])
```
